### ISSUE TITLE
Feature suggestion: Animated backgrounds

### ISSUE THREAD

**Issue by hexahigh** [NickvisionApps/Cavalier#152]
Hey, i love this app but there is one feature i feel is missing, which is animated backgrounds.
It would be nice to be able to use webp, avif, gif or other video formats as the background. And maybe a few settings to go along with it like controlling the playback speed.

**Comment by hexahigh** [comment]
Im not good at C# but it seems like the library that cavalier is using for rendering images does not support animated pictures

**Comment by nlogozzo** [comment]
Yeah I have to look into it. I believe we just use gtk to render the background, so it might be able to render the animated backgrounds.

But Cavalier will soon be rewritten in C++, so I have to see once I do that.
